index.js

```diff
@@ -22,7 +27,16 @@ module.exports = {
 
   init: function() {
     var bowerDeps = this.project.bowerDependencies();
+
     var VersionChecker = require('ember-cli-version-checker');
+    var options = this.options = this.options || {};
```

Comment:
Also, it is used just below (via `options.babel` which is also the same object as `this.options.babel`).